sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -757,6 +760,10 @@ CosmosDiagnosticsThresholds getEffectiveDiagnosticsThresholds(
         return clientLevelThresholds != null ? clientLevelThresholds : new CosmosDiagnosticsThresholds();
     }
 
+    CosmosEndToEndOperationLatencyPolicyConfig getEndToEndOperationLatencyPolicyConfig() {
```

Comment:
is this used anywhere?

Reply:
No. But its a getter for users to see what policy they set on the client. Not required?